assets/base.css

```diff
@@ -1191,37 +1191,71 @@ deferred-media {
   position: relative;
```

Comment:
The icon isnt looking centered. Is that expected? https://screenshot.click/09-44-7zefu-lvh2t.png

Reply:
Got it properly centered now. The svg was missing a viewbox property now I can scale it the way I want! 👍